backoff.go

```diff
@@ -41,7 +41,7 @@ type Backoff struct {
 	// with Max it is used to produce the "bounded wait duration".
 	Min time.Duration
 
-	// Min is a upper bound for the "transformed wait duration", in combination
+	// Min is an upper bound for the "transformed wait duration", in combination
```

Comment:
Are you sure this isn't supposed to be **Max**?
```suggestion
	// Max is an upper bound for the "transformed wait duration", in combination
```

Reply:
I am sure that I didn't even notice that bit!